exams/utils.py

```diff
@@ -59,3 +65,59 @@ def authorize_for_exam(mmtrack, course_run):
                 mmtrack.user.username,
                 course_run.edx_course_key
             )
+        else:
+            log.info(
+                'Unable to authorize user "%s" for exam, course id "%s"',
+                mmtrack.user.username,
+                course_run.edx_course_key
+            )
+
+
+def bulk_authorize_for_exam(program_id=None, username=None):
+    """
+    Authorize user(s) for exam(s).
+
+    Args:
+        program_id(str): Program id (optional) if you want authorization on specific program exams
+        username(str): User name (optional) whom you want to authorize for exam(s)
+    """
+    programs = Program.objects.filter(
+        live=True
+    ).exclude(exam_series_code__isnull=True).exclude(exam_series_code__exact="")
+    if program_id:
+        programs = programs.filter(id=program_id)
+
+    if not programs.exists():
+        log.info('[authorization_user_exam] Program(s) are not available for exam authorization.')
+        return
+
+    for program in programs:
+        program_enrollment_qset = ProgramEnrollment.objects.filter(program=program)
+        if username:
+            program_enrollment_qset = program_enrollment_qset.filter(user__username=username)
+
+        if not program_enrollment_qset.exists():
+            log.info('[authorization_user_exam] User(s) not exist for exam authorization.')
+            continue
+
+        for program_enrollment in program_enrollment_qset:
+            user = program_enrollment.user
+            mmtrack = get_mmtrack(user, program)
+            course_ids = set()
+
+            for edx_course_key in mmtrack.edx_key_course_map.keys():
+                course_ids.add(mmtrack.edx_key_course_map[edx_course_key])
```

Comment:
cannot you just do
`course_ids = set(mmtrack.edx_key_course_map.values())`
or am I missing something?